Sources/Helpers/OAAppSettings.h

```diff
@@ -356,6 +356,7 @@ typedef NS_ENUM(NSInteger, EOARulerWidgetMode)
 @property (assign, nonatomic) BOOL mapSettingShowFavorites;
 @property (assign, nonatomic) BOOL mapSettingShowOfflineEdits;
 @property (assign, nonatomic) BOOL mapSettingShowOnlineNotes;
+@property (assign, nonatomic) BOOL mapSettingShowOpacitySlider;
```

Comment:
Two sliders may be visible (overlay and underlay). Thus two variables are needed.

Reply:
Done.